Makefile

```diff
@@ -101,3 +101,7 @@ toggle-cw-killswitch:
 teardown:
 	@./scripts/babylon-integration/teardown.sh
 .PHONY: teardown
+
+proxy-setup:
```

Comment:
maybe we can add another make command to restart nginx server (for debugging)

Reply:
that's just `sudo systemctl reload nginx.service`. feel too small to be it's own make command